### DISCUSSION TITLE
Silent unpacking

### DISCUSSION THREAD

**Discussion by sov44** [sumatrapdfreader/sumatrapdf#3895] [Q&A]
Can you tell me how to unpack the installer without showing the process in the console?
`TempZ\Upd\SumatraPDF-3.5.2-64-install -s -x -d TempZ\mak\x64`
Shows the files to be extracted. (((

**Comment by kjk** [comment]
You can redirect the output to nothing e.g. in powershell add `| Out-Null`

`-s` option is to eliminate GUI, not logging to stdout.
